IDE/STM32Cube/wolfssl_example.c

```diff
@@ -1468,25 +1469,264 @@ static void PrintMemStats(void)
 #endif
 }
 
-#if 0
-static void* wolfMallocCb(size_t size)
+/* UART DMA IO Routines */
+#ifndef B115200
+#define B115200 115200
+#endif
+
+/* Max buffer for a single TLS frame */
+#ifndef MAX_RECORD_SIZE
+#define MAX_RECORD_SIZE (16 * 1024)
+#endif
+
+typedef struct {
+	uint16_t curr_index;
+        uint16_t data_len;
+	char buf[MAX_RECORD_SIZE];
+} tls13_buf;
+
+/* This sets which UART to do the TLS 1.3 connection over.  It is something you
+ * will have to configure in STMCubeIDE and then change here. */
+#define TLS_UART huart2
```

Comment:
Please add an `#ifndef TLS_UART` around this.

Reply:
Fixed in next commit.